packages/hyperion-autologging/src/ALType.ts

```diff
@@ -9,6 +9,7 @@ import { ALFlowlet, ALFlowletManager } from './ALFlowletManager';
 
 export type ALFlowletEvent = Readonly<{
   flowlet: ALFlowlet;
+  alflowlet?: ALFlowlet;
```

Comment:
nit: for consistency 

```suggestion
  alFlowlet?: ALFlowlet;
```